### PR TITLE
Https bg

### PR DESCRIPTION
**Pull request by DanGould**
@rancerun To serve from https, we should serve the image from our own server. Before we were always pulling from /wp-content/ on the wordpress site, now we pull from the public folder. Also compressed the background image to 120Kb because it was a 1.4Mb download before

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -35,7 +35,7 @@
         </div>
       </nav>
 
-      <section class="hero-section d-flex flex-grow-1 align-items-center justify-content-center text-light" style="background-image: url(http://citizensgov.org/wp-content/uploads/Womens-March-0001-1200x801.jpg)">
+      <section class="hero-section d-flex flex-grow-1 align-items-center justify-content-center text-light" style="background: url(./Womens-March-0001-1200x801.jpg) no-repeat fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;">
```
Comment (DanGould) [inline app/views/layouts/application.html.erb]: This could be improved by putting all `style` elements inside a class called .home-bg

Comment (rancerun) [reply app/views/layouts/application.html.erb]: i moved the css to the stylesheet, and i also moved the image location to asset/image to align with rails helper methods. should be ready to merge after your review

File: app/assets/stylesheets/landing_page.css.scss [hunk 2]
```diff
@@ -3,6 +3,14 @@ main {
   min-height: 100vh;
 }
 
+.homes-bg {
+	background: url('Womens-March-0001-1200x801.jpg') no-repeat fixed;
```
Comment (DanGould) [inline app/assets/stylesheets/landing_page.css.scss]: Have you tested this? Usually we prefix with `./` to make it obvious the url is relative to our current location

Comment (rancerun) [reply app/assets/stylesheets/landing_page.css.scss]: from what i can tell it's functional. i think it's a rails specific syntax

https://guides.rubyonrails.org/asset_pipeline.html#css-and-sass

**Pull request merged**